.circleci/config.yml.j2

```diff
@@ -32,44 +32,19 @@ jobs:
       - run:
           name: Release the Astronomer chart
           command: |
-            printf "%s" $${GCP_TOKEN} > /tmp/gcs_token.json
-            gcloud auth activate-service-account --key-file=/tmp/gcs_token.json
```

Comment:
looking at the changset for these two lines I don't understand. gcloud auth stuff on one side and setting py environment on the other. For my own knowledge could you explain this one?

Reply:
This was actually changed a month ago in #547 but only half merged, and now second half here. It's unneeded now